apps/projects/app/components/Card/Issue.js

```diff
@@ -105,15 +105,21 @@ const Issue = ({workStatus, title, repo, number, labels, isSelected, onClick, on
       }
       {workStatus !== undefined &&
         <ContextMenu>
-          <ContextMenuItem onClick={onSubmitWork}>
-            <ActionLabel>Submit Work</ActionLabel>
-          </ContextMenuItem>
-          <ContextMenuItem onClick={onRequestAssignment}>
-            <ActionLabel>Request Assignment</ActionLabel>
-          </ContextMenuItem>
-          <ContextMenuItem onClick={onReviewApplication}>
-            <ActionLabel>Review Application</ActionLabel>
-          </ContextMenuItem>
+          {workStatus === 'submit-work' || workStatus === 'review-work' &&
```

Comment:
I think the workstatus here is causing no options. I'm guessing it's because the logic isn't grouped here